lib/src/l10n/languages/croppy_pt.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:croppy/src/l10n/croppy_localizations.dart';
+
+class CroppyLocalizationsPt extends CroppyLocalizations {
+  CroppyLocalizationsPt() : super('pt');
+
+  @override
+  String get cancelLabel => 'Cancelar';
+
+  @override
+  String get cupertinoFreeformAspectRatioLabel => 'LIVRE';
+
+  @override
+  String get cupertinoOriginalAspectRatioLabel => 'ORIGINAL';
+
+  @override
+  String get cupertinoResetLabel => 'REINICIAR';
+
+  @override
+  String get cupertinoSquareAspectRatioLabel => 'QUADRADO';
+
+  @override
+  String get doneLabel => 'OK';
+
+  @override
+  String get materialFreeformAspectRatioLabel => 'Livre';
+
+  @override
+  String materialGetFlipLabel(LocalizationDirection direction) =>
+      'Flip to ${direction == LocalizationDirection.vertical ? 'vertical' : 'horizontal'}';
```

Comment:
Seems to still be in English

Reply:
Nop. Vertical and Horizontal are also portuguese words.

Reply:
I've changed the file, but not sure how to resolve it. 

https://github.com/kekland/croppy/pull/11/commits/ae86f9f8415af07bc9dd9aade521cf345640f271